hack/verify-import-aliases.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE[0]}")/..
+cd "${SCRIPT_ROOT}"
+
+ROOT_PATH=$(pwd)
+GOPATH=$(go env GOPATH | awk -F ':' '{print $1}')
+IMPORT_ALIASES="-import-aliases $ROOT_PATH/hack/.import-aliases"
+INCLUDE_PATH="-include-path (cmd|pkg|test/e2e|test/helper)"
+CONFIRM="-confirm"
+
+ret=0
+read -p  "Do you want to update file automatically [y/n] " choice
+case ${choice} in
+  y|Y)
+      go run "$GOPATH"/pkg/mod/k8s.io/kubernetes@v1.21.3/cmd/preferredimports/preferredimports.go ${IMPORT_ALIASES} ${INCLUDE_PATH} ${CONFIRM} ${ROOT_PATH}
```

Comment:
Shell check suggests: Double quote to prevent globbing and word splitting [SC2086](https://github.com/koalaman/shellcheck/wiki/SC2086) like this:
```
go run "$GOPATH"/pkg/mod/k8s.io/kubernetes@v1.21.3/cmd/preferredimports/preferredimports.go "${IMPORT_ALIASES}" "${INCLUDE_PATH}" "${CONFIRM}" "${ROOT_PATH}"
```

Reply:
The dir "kubernetes@v1.21.3" may not work when the version of Kubernetes update.

Reply:
Can we install the `preferredimports` command first and then run it?

Reply:
> Can we install the `preferredimports` command first and then run it?

OK.